app/routes/articles.js

```diff
@@ -55,6 +60,8 @@ router.post('/', isLoggedIn, hasAuthorRole, (req, res) => {
             req.flash('error', 'Oops! Something went wrong!')
             return res.redirect('/')
         }
+
+        sendNewsletters(article).catch(err => req.log('SendNewsletters in CREATE', err))
```

Comment:
Shouldn't you receive newsletter when an article is being approved, instead of created?

Reply:
Good catch